libpod/runtime.go

```diff
@@ -704,7 +704,7 @@ var libimageEventsMap = map[libimage.EventType]events.Status{
 	libimage.EventTypeImageUnmount: events.Unmount,
 }
 
-// libimageEvents spawns a goroutine in the background which is listenting for
+// libimageEvents spawns a goroutine in the background which is listening for
 // events on the libimage.Runtime.  The gourtine will be cleaned up implicitly
```

Comment:
Might as well fix this one ("gourtine") too, no? How did it slip past? (insert comment about French-Canadian glopmixes of frites and gravy).

FWIW I think the sentence would read better as "...spawns a goroutine which will listen for events in the background..." (or even skipping the "in the background", since that feels implicit).

Reply:
I tried to change according to your suggestion. Does it look ok now?